nixos/modules/services/system/nscd.nix

```diff
@@ -38,6 +38,11 @@ in
         default = if pkgs.stdenv.hostPlatform.libc == "glibc"
           then pkgs.stdenv.cc.libc.bin
           else pkgs.glibc.bin;
+        defaultText = literalExample ''
```

Comment:
`literalExample` is deprecated, opened https://github.com/NixOS/nixpkgs/pull/170412